package.json

```diff
@@ -156,6 +156,7 @@
     "chai-as-promised": "^7.1.1",
     "cross-env": "^7.0.2",
     "deep-object-diff": "^1.1.0",
+    "diff": "^4.0.2",
```

Comment:
`diff` was actually already an indirect dependency found in the shrinkwrap file. This makes it a direct dev dependency.